src/server_manager/web_app/ui_components/outline-gcp-info-setting.ts

```diff
@@ -0,0 +1,53 @@
+/*
+  Copyright 2021 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import '@polymer/paper-input/paper-input';
+
+import {customElement, html, LitElement, property} from 'lit-element';
+
+import {COMMON_STYLES} from './cloud-install-styles';
+import {CloudLocation} from '../../model/location';
+import {getShortName} from '../location_formatting';
+import {SETTINGS_STYLES} from './outline-server-settings-styles';
+
+/** Setting card for displaying GCP-specific info */
+@customElement('outline-gcp-info-setting')
+export class OutlineGcpInfoSetting extends LitElement {
```

Comment:
I don't understand. Why do you need this element? Can't you reuse the DO one and pass the name and icon as input?

Reply:
OK, done.